### PR TITLE
Uninstall gem 'forest_liana'/Forest Admin

### LINKED ISSUE #323: Uninstall Forest Admin
# Feature
## Why is this feature being added?

This PR added Forest admin: https://github.com/OperationCode/operationcode_backend/pull/283

It needs to be uninstalled, as we are using ActiveAdmin instead.  Here are the instructions to uninstall forestadmin:

https://doc.forestadmin.com/knowledge-base.html#uninstall
## What should your feature do?

- [ ] Clean uninstall of ForestAdmin and everything from https://github.com/OperationCode/operationcode_backend/pull/283

### PR DESCRIPTION
**Pull request by kirbygit**
# Description of changes

- Completely uninstall gem 'forest_liana' dependencies
- removed Forest routes in config/routes.rb
- removed forest_liana.rb in initializers
- run `make bundle` to update dependencies

# Issue Resolved

Fixes #323 reference #283

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -50,8 +50,6 @@ GEM
     airtable (0.0.9)
       httparty
     arel (7.1.4)
-    arel-helpers (2.6.1)
```
Comment (hpjaj) [inline Gemfile.lock]: Try re-running `make bundle` to see if that resolves your gemfile.lock issues.

If not, try to rebase on master, then `make bundle`.

File: config/initializers/forest_liana.rb [hunk 2]
```diff
@@ -1,2 +0,0 @@
-ForestLiana.env_secret = Rails.application.secrets.forest_env_secret
```
Comment (hpjaj) [inline config/initializers/forest_liana.rb]: This file needs to be deleted, not just the contents removed.  I can't tell if it has been removed, or just emptied out.

**Comment by hpjaj** [comment]
In addition to the changes in here, can you pls change the PR's title to something closer to the title of the associated issue.

We want these PR titles to summarize all of the changes in the PR. Moreover, when you look back and read through the list of closed PR's, you should be able to tell exactly what PR is doing, from its title.

This is very helpful later on, when you are trying to debug an issue, or determine how to approach an upcoming feature, etc.

File: Gemfile.lock (hunk 1 above)
Comment (kirbygit) [reply Gemfile.lock]: hmm it doesn't resolve the conflict issues let me just try to make another branch and do all the proper changes together

File: config/initializers/forest_liana.rb (hunk 2 above)
Comment (kirbygit) [reply config/initializers/forest_liana.rb]: ok I did not see this

**Comment by kirbygit** [comment]
I made another pr

**Pull request closed without merging**